libc/test/src/fenv/exception_flags_test.cpp

```diff
@@ -48,6 +54,8 @@ TEST(LlvmLibcFenvTest, GetExceptFlagAndSetExceptFlag) {
   LIBC_NAMESPACE::fputil::raise_except(FE_INVALID);
   fexcept_t eflags;
   LIBC_NAMESPACE::fegetexceptflag(&eflags, FE_ALL_EXCEPT);
+  ASSERT_EQ(LIBC_NAMESPACE::fetestexceptflag(&eflags, FE_ALL_EXCEPT),
+            FE_INVALID);
```

Comment:
Can you add one more `fetestexceptflag` after `fputil::raise_except(FE_OVERFLOW | FE_INEXACT)` that verifies that those 2 are set?

Reply:
Good idea!